src/React.purs

```diff
@@ -259,6 +259,18 @@ type ReactSpec props state eff =
   , componentWillUnmount :: ComponentWillUnmount props state eff
   }
 
+cmapSpec
+  :: forall props' props state eff
+   . (props -> props')
+  -> ReactSpec props' state eff
+  -> ReactSpec props Unit eff
+cmapSpec f sp = (spec unit renderFn) { displayName = sp.displayName <> "Mapped" }
```

Comment:
Also, calling `spec` like this loses information if the `Spec` wasn't already built with just `spec`, right? For example, lifecycle methods will be lost.